### ISSUE TITLE
Charleston: Missing streets and number (us/sc/charleston.json)

### ISSUE THREAD

**Issue by Joxit** [openaddresses/openaddresses#6106]
Hi there,

It seems that since March 11, 2022 number and streets are missing in [Charleston](https://github.com/openaddresses/openaddresses/blob/master/sources/us/sc/charleston.json).

They change the data structruce, now an object looks like:
```
OBJECTID: 13636
POSTAL_TOWN: ISLE OF PALMS
POSTAL_STATE: SC
POSTAL_CODE: 29451
PID: 6040900083
WHOLE_ADDRESS: 1 DOLPHIN ROW
Point:
X: 2384224.3622047305
Y: 354783.50524935126 
```

![image](https://user-images.githubusercontent.com/5153882/207288556-2dc55841-8b65-4052-bd5b-cd8f79e9638f.png)

See history: https://batch.openaddresses.io/data/1841/history

**Comment by iandees** [comment]
I just kicked off another build for this source. Let's see if it helps.

**Comment by Joxit** [comment]
Fixed with #6111 thank you :grin:
